OCABundles/schema/bcgov-digital-trust/member-card/README.md

```diff
@@ -9,9 +9,8 @@ A verifiable credential schema for a lawyer's membership card that used in a dem
 
 | Identifier                                 | Location  | URL                                                   |
 | ------------------------------------------ | --------- | ----------------------------------------------------- |
-| 63ZiwyeZeazA6AhYRYm2zD:2:Member Card:1.5.1 | candy:dev | https://candyscan.idlab.org/tx/CANDY_DEV/domain/26676 |
-| Trx3R1frdEzbn34Sp1jyX:2:Member Card:1.5.1  | candy:dev | https://candyscan.idlab.org/tx/CANDY_DEV/domain/26514 |
-| XUxBrVSALWHLeycAUhrNr9:2:Member Card:1.5.1 | candy:dev | https://candyscan.idlab.org/tx/CANDY_DEV/domain/27369 |
+| AuJrigKQGRLJajKAebTgWu:3:CL:209526:default | sovrn:staging | https://sovrin.org/ |
+| 4xE68b6S5VRFrKMMG1U95M:3:CL:59232:default | sovrn:mainnet | https://sovrin.org/ |
```

Comment:
Same typo, and the link should be: https://indyscan.io/tx/SOVRIN_MAINNET/domain/59233